packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -322,7 +318,39 @@ class ServiceConnectionManager {
       throw Exception(message);
     }
 
-    final viewId = flutterView['id'];
+    return flutterView['id'];
+  }
+
+  /// Flutter engine returns estimate how much memory is used by layer/picture raster
+  /// cache entries in bytes.
+  ///
+  /// Call to returns JSON payload 'EstimateRasterCacheMemory' with two entries:
+  ///   layerBytes - layer raster cache entries in bytes
+  ///   pictureBytes - picture raster cache entries in bytes
+  Future<Response> getRasterCacheMetrics() async {
```

Comment:
we should avoid method names that start with get. This one can be `rasterCacheMetrics()` I don't know what to do for `getDisplayRefreshRate` as `DisplayRefreshRate` makes it sound like we are displaying the refresh rate. Maybe `queryDisplayRefreshRate` or similar? Avoiding `get` is a dart style guide suggestion we didn't follow that well as some of this code was ported from Java.

Reply:
There are a few methods that are getters besides these, I'll change all of them in this file e.g., getFlutterVersion, getAdbMemoryInfo, getFlutterViewId.